test/src/test/java/org/corfudb/infrastructure/log/StreamLogCompactionTest.java

```diff
@@ -0,0 +1,53 @@
+package org.corfudb.infrastructure.log;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.AbstractCorfuTest;
+import org.corfudb.infrastructure.ServerContext;
+import org.corfudb.infrastructure.ServerContextBuilder;
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.junit.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+
+@Slf4j
+public class StreamLogCompactionTest {
```

Comment:
See other test classes and extend from appropriate Base class to have a uniform output.